### ISSUE TITLE
List of models empty after refreshing the models page

### ISSUE THREAD

**Issue by viennem** [ditrit/leto-modelizer#189]
**Describe the bug**

On the models page, if you have existing models in the list of models and you refresh the page, the models are not displayed after the refresh.
Going back to the home page and reopening the project correctly populates the list of models.

**To Reproduce**

1. Create a project and a model.
2. Go back to the models page.
3. Refresh the page.
4. Notice that the list of models is now empty.
5. Go to the home page, then reopen your project.
6. Notice that the list of models is not empty anymore.

**Screenshots**

![image](https://user-images.githubusercontent.com/20184007/219623711-8a3a650f-6f82-4376-9ce0-839d1422b6f9.png)

**Application:**

- leto version [1.0.0]

**Comment by viennem** [comment]
Reopening, because I noticed that the issue happens again.
However, it does not happen every time, but you can try to refresh a few times to reproduce it.

**Comment by viennem** [comment]
I added connsole logs in the following functions:

* `PluginManager#initPlugins()`:
   ```js
   export async function initPlugins() {
     console.log('1. initPlugins: plugins', plugins);
     return Promise.allSettled(Object.keys(plugins).map(instantiatePlugin))
       .then((allresults) => {
         console.log('2. initPlugins: allresults', allresults);
         instanciatePlugins = allresults
           .filter((r) => r.status === 'fulfilled')
           .map((r) => r.value);
       });
   }
   ```
* `Project#getAllModels()`:
   ```js
   export async function getAllModels(projectId) {
     const path = process.env.MODELS_DEFAULT_FOLDER !== ''
       ? `${projectId}/${process.env.MODELS_DEFAULT_FOLDER}`
       : `${projectId}`;
     const plugins = getPlugins();
     console.log('3. getAllModels: plugins', plugins);

     return Promise.allSettled(
       plugins.map(({ data }) => getPluginModels(path, data.name)),
     ).then((results) => results
       .filter((result) => result.status === 'fulfilled')
       .map((result) => result.value)
       .flat());
   }
   ```

Plugins must all be initialized before `Project#getAllModels()` is called by the `DiagramsCard` in the `ModelsPage`.

However, `PluginManager#initPlugins()` and `Project#getAllModels()` are called in parallel in separate `onMounted()` functions (in `App.vue` and `DiagramsCard.vue`).

Here is a video showing that the console logs are not in the expected order when this bug happens:

https://github.com/ditrit/leto-modelizer/assets/20184007/941f1401-bfa2-4425-b405-8ac3b4e82469

**Comment by viennem** [comment]
Fixed by #366.
